CMakeLists.txt

```diff
@@ -133,6 +134,10 @@ if(CMAKE_BUILD_TYPE MATCHES Release)
   endif()
 endif()
 
+if (COVERAGE)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftest-coverage")
```

Comment:
Ah, and `-fprofile-arcs` for storing arcs between basic blocks in the data flow graph.